### PR TITLE
Add the diagnostics settings

### PR DESCRIPTION
**Pull request by hyejin672**
Adding the diagnostics setting to the MCVP.

### CODE REVIEW

**Review by mpwillia — APPROVED** [review]
I understand the need for the second ARM template. Maybe we can find a single ARM template solution in the future.

File: deployment/ConnectedVehicleARMTemplate.json [hunk 1]
```diff
@@ -74,6 +74,28 @@
...
+                "pergb2018",
+                "Free",
+                "Standalone",
+                "PerNode",
+                "Standard",
+                "Premium"
+            ],
+            "defaultValue": "pergb2018",
+            "metadata": {
+                "description": "Pricing tier: PerGB2018 or legacy tiers (Free, Standalone, PerNode, Standard or Premium) which are not available to all customers."
```
Comment (explorerG) [inline deployment/ConnectedVehicleARMTemplate.json:97]: The description confused me. What is not available to the customers?

Comment (hyejin672) [reply deployment/ConnectedVehicleARMTemplate.json:97]: Good point. This was copied over from the sample template. I will update it.

Comment (hyejin672) [reply deployment/ConnectedVehicleARMTemplate.json]: Confirmed all in the list is available. 

https://docs.microsoft.com/en-us/azure/templates/microsoft.operationalinsights/2020-03-01-preview/workspaces?tabs=bicep#workspacesku

**Review by explorerG — APPROVED** [review]
Assuming the templates are tested for functionality

**Pull request merged**